inc/compat/themes/compat-theme-Divi.php

```diff
@@ -50,7 +50,7 @@ public function output_checkout_header_logo() {
 
             ob_start();
             ?>
-                <a href="<?php echo esc_url( home_url( '/' ) ); ?>">
+                <a href="<?php echo apply_filters( 'fc_checkout_header_logo_home_url', esc_url( home_url( '/' ) ) ); ?>">
```

Comment:
@alamgircsebd the value returned from the filter `fc_checkout_header_logo_home_url` should be passed 
 to the `esc_url` function prior to outputting it to the HTML to avoid security issues.

If the filter `fc_checkout_header_logo_home_url ` returns an invalid or malicious string, the `esc_url` function would sanitize it.